packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -214,6 +214,11 @@ export interface IGCRuntimeOptions {
      */
     runSweep?: boolean;
 
+    /**
+     * Sets the session expiry for containers when GC sweep is enabled
+     */
+    gcSessionExpiryTime?: number;
```

Comment:
Don't add this option publicly. We don't want anyone controlling the session timeout. You can still use it - just don't add it here.